runtime/autoload/man.vim

```diff
@@ -116,7 +116,11 @@ function! s:get_page(path) abort
   " Force MANPAGER=cat to ensure Vim is not recursively invoked (by man-db).
   " http://comments.gmane.org/gmane.editors.vim.devel/29085
   " Respect $MANWIDTH, or default to window width.
-  return s:system(['env', 'MANPAGER=cat', (empty($MANWIDTH) ? 'MANWIDTH='.winwidth(0) : ''), 'man', a:path])
```

Comment:
Yes, this is actually a lot nicer :) Sorry, I really have no experience at all in vimL.

Shall I update the pull request or is it too trivial a change?